Formula/elixir.rb

```diff
@@ -30,6 +31,6 @@ def install
   end
 
   test do
-    system "#{bin}/elixir", "-v"
+    assert_match(%r{(compiled with Erlang/OTP 25)}, shell_output("#{bin}/elixir -v"))
```

Comment:
Coming from #116834, I think this hardcoded value is not a good test. erlang dependency would be changed over time.